server/lib/genome/importers/file_importers/drugbank.rb

```diff
@@ -0,0 +1,246 @@
+module Genome; module Importers; module FileImporters; module Drugbank;
+  class Importer < Genome::Importers::Base
+    attr_reader :file_path
+    attr_reader :source_db_name
+    attr_reader :source
+    attr_reader :drug_filter
+
+    def initialize(file_path)
+      @file_path = file_path
+      @source_db_name = 'DrugBank'
+    end
+
+    def run_parser
+      @drug_filter = DrugFilter.new
+      parser = Nokogiri::XML::SAX::Parser.new(drug_filter)
+      parser.parse(File.open(file_path))
+    end
+
+    def create_new_source
+      @source ||= Source.create(
+        {
+            base_url: 'https://go.drugbank.com/',
```

Comment:
base_url should be `https://go.drugbank.com/drugs/` to be consistent w/ other sources